Regular expression for matching black exe version is not valid for old black release

The regular expression was modified in #1178 to match the new version format, but no longer works for the previous format.

```
10:29:19  [ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.22.4:apply (default) on project buildtools: Execution default of goal com.diffplug.spotless:spotless-maven-plugin:2.22.4:apply failed: You specified version 19.10b0, but Spotless found version
10:29:19  [ERROR] Try running {@code pip install --force-reinstall black==19.10b0}, or else specify {@code black('version')} to Spotless
10:29:19  [ERROR]   github issue to handle this better: https://github.com/diffplug/spotless/issues/674
10:29:19  [ERROR] > arguments: [/usr/local/bin/black, --version]
10:29:19  [ERROR] > exit code: 0
10:29:19  [ERROR] >    stdout: black, version 19.10b0
10:29:19  [ERROR] >    stderr: (empty)
10:29:19  [ERROR] -> [Help 1]
```

- Apache Maven 3.8.4
- spotless-maven-plugin 2.22.4

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -10,6 +10,8 @@ This document is intended for Spotless developers.
 We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (starting after version `1.27.0`).
 
 ## [Unreleased]
+### Fixed
+* Update the `black` version regex to fix `19.10b0` and earlier. (fixes [#1195](https://github.com/diffplug/spotless/issues/1195), regression introduced in `2.25.0`)
 
 ## [2.25.2] - 2022-05-03
 ### Changes

--- a/lib/src/main/java/com/diffplug/spotless/python/BlackStep.java
+++ b/lib/src/main/java/com/diffplug/spotless/python/BlackStep.java
@@ -64,7 +64,7 @@ public class BlackStep {
 		String trackingIssue = "\n  github issue to handle this better: https://github.com/diffplug/spotless/issues/674";
 		String exeAbsPath = ForeignExe.nameAndVersion("black", version)
 				.pathToExe(pathToExe)
-				.versionRegex(Pattern.compile("(?:black,|version) (\\S*)"))
+				.versionRegex(Pattern.compile("(?:black, version|black,|version) (\\S*)"))
 				.fixCantFind("Try running {@code pip install black=={version}}, or else tell Spotless where it is with {@code black().pathToExe('path/to/executable')}" + trackingIssue)
 				.fixWrongVersion("Try running {@code pip install --force-reinstall black=={version}}, or else specify {@code black('{versionFound}')} to Spotless" + trackingIssue)
 				.confirmVersionAndGetAbsolutePath();

--- a/plugin-gradle/CHANGES.md
+++ b/plugin-gradle/CHANGES.md
@@ -3,6 +3,8 @@
 We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (starting after version `3.27.0`).
 
 ## [Unreleased]
+### Fixed
+* Update the `black` version regex to fix `19.10b0` and earlier. (fixes [#1195](https://github.com/diffplug/spotless/issues/1195), regression introduced in `6.5.0`)
 
 ## [6.5.2] - 2022-05-03
 ### Changes

--- a/plugin-maven/CHANGES.md
+++ b/plugin-maven/CHANGES.md
@@ -3,6 +3,8 @@
 We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (starting after version `1.27.0`).
 
 ## [Unreleased]
+### Fixed
+* Update the `black` version regex to fix `19.10b0` and earlier. (fixes [#1195](https://github.com/diffplug/spotless/issues/1195), regression introduced in `2.22.2`)
 
 ## [2.22.4] - 2022-05-03
 ### Changes

